templates/directory.php

```diff
@@ -426,21 +427,23 @@ function pmpromd_shortcode($atts, $content=null, $code="")
 			{
 				foreach($theusers as $auser):
 					$auser = get_userdata($auser->ID);
+					var_dump($auser);
 					$auser->membership_level = pmpro_getMembershipLevelForUser($auser->ID);
+					$user_identifier = apply_filters( 'pmpromd_user_identifier', 'user_nicename' );
```

Comment:
This is used in multiple places, let's make a new function that has the hook in that and call that function here.